app/models/rental.rb

```diff
@@ -130,7 +130,7 @@ def sum_amount
   attr_accessor :skip_reservation_validation
 
   def create_financial_transaction
-    rental_amount = (((end_time.to_date - start_time.to_date).to_i - 1) * item_type.fee_per_day) + item_type.base_fee
+    rental_amount = ((end_time.to_date - start_time.to_date).to_i * item_type.fee_per_day) + item_type.base_fee
```

Comment:
I would love to see some tests of the rental model that ensures the correctness of your change.

A (hypothetical) zero day rental = 0? Error?
A one day rental with a base fee of `A` and a per day fee of `B` = X rental cost
A two day rental with a base fee of `A` and a per day fee of `B` = Y rental cost